protos/merlin/transformer/spec/table.proto

```diff
@@ -0,0 +1,82 @@
+syntax = "proto3";
+
+package merlin.transformer;
+
+import "transformer/spec/common.proto";
+
+option go_package = "github.com/gojek/merlin/pkg/transformer/spec";
+
+
+message Table {
+  string name = 1;
+  BaseTable baseTable = 2;
+  repeated Column columns = 3;
+}
+
+message BaseTable {
+  oneof baseTable {
+    FromJson fromJson = 1;
+    FromTable fromTable = 2;
+  }
+}
+
+message Column {
+  string name = 1;
+  oneof columnValue {
+    FromJson fromJson = 2;
+    string expression = 3;
+  }
+}
+
+message TableTransformation {
+  string inputTable = 1;
+  string outputTable = 2;
+  repeated TransformationStep steps = 3;
+}
+
+message TransformationStep {
+  repeated string dropColumns = 1;
+  repeated SortColumnRule sort = 2;
+  RenameColumn renameColumn = 3;
+  AddOrUpdateColumn addOrUpdateColumn = 4;
```

Comment:
It could be, this proto is only placeholder for now, it might change during implementation. @tiopramayudi please take note on this, in case you are the one implementing.